### PR TITLE
Add roadmap

### PR DESCRIPTION
**Pull request by tenmozes**
Fix some spells and typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ Cluster version is available [here](https://github.com/VictoriaMetrics/VictoriaM
   [Outperforms InfluxDB and TimescaleDB by up to 20x](https://medium.com/@valyala/measuring-vertical-scalability-for-time-series-databases-in-google-cloud-92550d78d8ae).
 * [Uses 10x less RAM than InfluxDB](https://medium.com/@valyala/insert-benchmarks-with-inch-influxdb-vs-victoriametrics-e31a41ae2893) when working with millions of unique time series (aka high cardinality).
 * High data compression, so [up to 70x more data points](https://medium.com/@valyala/when-size-matters-benchmarking-victoriametrics-vs-timescale-and-influxdb-6035811952d4)
-  may be crammed into a limited storage comparing to TimescaleDB.
-* Optimized for storage with high-latency IO and low iops (HDD and network storage in AWS, Google Cloud, Microsoft Azure, etc). See [graphs from these benchmarks](https://medium.com/@valyala/high-cardinality-tsdb-benchmarks-victoriametrics-vs-timescaledb-vs-influxdb-13e6ee64dd6b).
+  maybe crammed into limited storage comparing to TimescaleDB.
```
Comment (hagen1778) [inline README.md]: `maybe` is an adverb. Plz consider following cases:
- Maybe you can store 70x times more datapoints comparing to TimescaleDB
- 70x times more datapoints may be stored comparing to TimescaleDB

File: README.md [hunk 2]
```diff
@@ -321,51 +322,51 @@ The `/api/v1/export` endpoint should return the following response:
...
+from sources. Building from sources is reasonable when developing additional features specific
 to your needs.
 
 
 #### Development build
 
 1. [Install Go](https://golang.org/doc/install). The minimum supported version is Go 1.12.
 2. Run `make victoria-metrics` from the root folder of the repository.
-   It will build `victoria-metrics` binary and put it into the `bin` folder.
+   It builds `victoria-metrics` binary and put it into the `bin` folder.
```
Comment (hagen1778) [inline README.md]: Should it be `and puts` then?

File: README.md [hunk 3]
```diff
@@ -321,51 +322,51 @@ The `/api/v1/export` endpoint should return the following response:
...
 
 1. [Install docker](https://docs.docker.com/install/).
 2. Run `make victoria-metrics-prod` from the root folder of the repository.
-   It will build `victoria-metrics-prod` binary and put it into the `bin` folder.
+   It builds `victoria-metrics-prod` binary and put it into the `bin` folder.
 
 #### ARM build
 
-ARM builds may run on Raspberry Pi or on [energy-efficient ARM servers](https://blog.cloudflare.com/arm-takes-wing/).
+ARM build may run on Raspberry Pi or on [energy-efficient ARM servers](https://blog.cloudflare.com/arm-takes-wing/).
```
Comment (hagen1778) [inline README.md]: It was plural because there could be multiple ARM builds

File: README.md [hunk 4]
```diff
@@ -476,15 +477,15 @@ Rough estimation of the required resources for ingestion path:
 
 * Network usage: outbound traffic is negligible. Ingress traffic is ~100 bytes per ingested data point via
   [Prometheus remote_write API](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
-  The actual ingress bandwitdh usage depends on the average number of labels per ingested metric and the average size
-  of label values. Higher number of per-metric lables and longer label values mean higher ingress bandwidth.
+  The actual ingress bandwidth usage depends on the average number of labels per ingested metric and the average size
+  of label values. The higher number of per-metric labels and longer label values mean higher ingress bandwidth.
```
Comment (hagen1778) [inline README.md]: `the` transforms construction into comparison like `the more ... the more` but without second part. It should be `the higher number ..., the higher ingress...`

File: README.md [hunk 5]
```diff
@@ -623,10 +624,23 @@ The most interesting metrics are:
 
 * If VictoriaMetrics doesn't work because of certain parts are corrupted due to disk errors,
   then just remove directoreis with broken parts. This will recover VictoriaMetrics at the cost
-  of data loss stored in the broken parts. In the future `vmrecover` tool will be created
+  of data loss stored in the broken parts. In the future, `vmrecover` tool will be created
   for automatic recovering from such errors.
 
+## Roadmap
 
+The road to 2.0 version:
```
Comment (hagen1778) [inline README.md]: version 2.0 means breaking changes. Are we sure about it?

File: README.md [hunk 6]
```diff
@@ -623,10 +624,23 @@ The most interesting metrics are:
 
 * If VictoriaMetrics doesn't work because of certain parts are corrupted due to disk errors,
   then just remove directoreis with broken parts. This will recover VictoriaMetrics at the cost
-  of data loss stored in the broken parts. In the future `vmrecover` tool will be created
+  of data loss stored in the broken parts. In the future, `vmrecover` tool will be created
   for automatic recovering from such errors.
 
+## Roadmap
```
Comment (hagen1778) [inline README.md]: Wouldn't it be better if we have roadmap in form of issue so users can subscribe to it?

Comment (tenmozes) [reply README.md]: we do

File: README.md (hunk 5 above)
Comment (tenmozes) [reply README.md]: I don't think that we can deliver these features without breaking changes, specially replication

File: README.md (hunk 3 above)
Comment (tenmozes) [reply README.md]: what this sentence for? Define the count or the ability

File: README.md [hunk 7]
```diff
@@ -623,10 +624,23 @@ The most interesting metrics are:
...
 
+The road to 2.0 version:
+
+- [ ] Support of Object Storages (GCS, S3, Azure Storage) [#38](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/38)
+- [ ] Data downsampling [#36](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/36)
+- [ ] Alert Manager Integration [#119](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/119) 
+- [ ] Replication [#118](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/118)
+- [ ] CLI tool for data migration, re-balancing and adding/removing nodes [#103](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/103)
+- [ ] String as account id [#102](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/102)
```
Comment (valyala) [inline README.md]: I don't think this is important feature to mention here

File: README.md [hunk 8]
```diff
@@ -623,10 +624,23 @@ The most interesting metrics are:
...
   for automatic recovering from such errors.
 
+## Roadmap
 
+The road to 2.0 version:
+
+- [ ] Support of Object Storages (GCS, S3, Azure Storage) [#38](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/38)
+- [ ] Data downsampling [#36](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/36)
+- [ ] Alert Manager Integration [#119](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/119) 
+- [ ] Replication [#118](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/118)
```
Comment (valyala) [inline README.md]: I'd move this to the top of the list.

File: README.md (hunk 3 above)
Comment (valyala) [reply README.md]: Let's leave `build` for the consistency sake with the rest of the text.

**Pull request merged**